src/routes/results/[results_string]/+page.svelte

```diff
@@ -114,7 +131,15 @@
 		{/each}
 		<path stroke-width="3" stroke="navy" fill="navy" opacity="0.8" d={drawAnswerShape(answers)} />
 		{#each formattedAnswers as ans}
-			<circle cx={ans.xCoord} cy={ans.yCoord} fill="navy" r={config.answerR}></circle>
+			<circle
```

Comment:
this should be resolved now! Svelte is particularly accessibility minded, so it didn't like that I was missing the focus and blur handler, as well as the ARIA role. I went with the button since it's kind of interactive with the hover, but I am very open to this changing if there's a more appropriate role to assign the label